src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBase.iOSAndroid.cs

```diff
@@ -59,10 +60,10 @@ private int PageSize
 			}
 			else
 			{
-				if (ScrollViewer?.Style == Uno.UI.GlobalStaticResources.ListViewBaseScrollViewerStyle)
+				if (ScrollViewer?.Style != null && ScrollViewer.Style == ResourceResolver.GetSystemResource<Style>("ListViewBaseScrollViewerStyle")) //TODO: this, too, properly
```

Comment:
Todo in this PR ?

Reply:
Depends if I end up adjusting the timing of template application to the Measure phase. I'd rather push that particular change back, but it depends on what regressions come up in testing.

Once templates are materialized on Measure, it'll be trivially easy to give ScrollViewers inside virtualized lists a 'fake' ScrollContentPresenter without any user intervention.